metrology/reporter/statsd.py

```diff
@@ -0,0 +1,112 @@
+import re
+import socket
+import sys
+
+from metrology.instruments import *  # noqa
```

Comment:
I don't understand this question. If you ask why I import all instruments - it's because reporter gets all data as instruments objects and reporter class need to know what are they and extract values from them correctly.